doc/guides/commit-queue.md

```diff
@@ -0,0 +1,103 @@
+# Commit Queue
+
+> tl;dr: You can land Pull Requests by adding the `commit-queue` label to it.
+> This feature is still experimental.
```

Comment:
Why is this a quote though?

Reply:
Just to show with different formatting, could also be bold or italic

Reply:
I think it's fine to just have this at the beginning without specific formatting.
As for the `This feature is still experimental.` I'd go with the formatting we have for docs.
So:

Pull Requests can be landed automatically by adding the `commit-queue` label to it.

> Stability: 1 - Experimental